### PR TITLE
Question card

### PR DESCRIPTION
**Pull request by tienlatien252**

### CODE REVIEW

File: frontend/src/components/QuestionCard/QuestionCard.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+import Card from "components/Card/Card.js";
+import CardBody from "components/Card/CardBody.js";
+import CardHeader from "components/Card/CardHeader.js";
+import Radio from "@material-ui/core/Radio";
+import FormControlLabel from "@material-ui/core/FormControlLabel";
+import FiberManualRecord from "@material-ui/icons/FiberManualRecord";
+import Grid from '@material-ui/core/Grid';
+
+import { makeStyles } from '@material-ui/core/styles';
+import radoStyles from "assets/jss/material-kit-react/customCheckboxRadioSwitch.js";
```
Comment (linhvovo) [inline frontend/src/components/QuestionCard/QuestionCard.js:16]: Typo: radioStyles

File: frontend/src/components/QuestionCard/QuestionCard.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    textAlign: "center"
+  },
+  textAlignLeft: {
+    textAlign: "left"
+  },
+  image: {
+    maxWidth: "100%",
+    height: "auto"
+  },
+  ...radoStyles
```
Comment (linhvovo) [inline frontend/src/components/QuestionCard/QuestionCard.js:30]: Typo: radioStyles

File: frontend/src/components/QuestionCard/QuestionCard.js [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+};
+
+const useStyles = makeStyles(styles);
+
+export default function QuestionCard(props) {
+  const classes = useStyles();
+  const { className, question_id, description, picture_link, answers, index } = props;
```
Comment (linhvovo) [inline frontend/src/components/QuestionCard/QuestionCard.js:37]: Console in Inspect Element says className and question_id are defined but never used.

File: frontend/src/components/QuestionCard/QuestionCard.js (hunk 1 above)
Comment (tienlatien252) [reply frontend/src/components/QuestionCard/QuestionCard.js]: Fix

File: frontend/src/components/QuestionCard/QuestionCard.js (hunk 2 above)
Comment (tienlatien252) [reply frontend/src/components/QuestionCard/QuestionCard.js]: Fixed

File: frontend/src/components/QuestionCard/QuestionCard.js (hunk 3 above)
Comment (tienlatien252) [reply frontend/src/components/QuestionCard/QuestionCard.js]: Deleted className but keep question_id because it might be useful when creating testing page

**Review by linhvovo — APPROVED** [review]

**Pull request merged**